### ISSUE TITLE
how to change save path "runs/segment/train0" Use your own file path such as /mypath/

### ISSUE THREAD

**Issue by minwim** [ultralytics/ultralytics#3822]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

WARNING ⚠️ Upgrade to torch>=2.0.0 for deterministic training.
yolo/engine/trainer: task=segment, mode=train, model=/home/disk/wlm/ultralytics/workspace/models/0712-asc1_2048-s-seg.pt, data=./yaml/infects-0529-sxyw.yaml, epochs=150, patience=150, batch=40, imgsz=2048, save=True, save_period=-1, cache=False, device=[0, 1, 2, 3, 4, 5, 6, 7], workers=56, project=None, name=None, exist_ok=False, pretrained=True, optimizer=auto, verbose=True, seed=0, deterministic=True, single_cls=False, rect=False, cos_lr=False, close_mosaic=0, resume=False, amp=True, fraction=1.0, profile=False, overlap_mask=True, mask_ratio=4, dropout=0.0, val=True, split=val, save_json=False, save_hybrid=False, conf=None, iou=0.7, max_det=300, half=False, dnn=False, plots=True, source=None, show=False, save_txt=False, save_conf=False, save_crop=False, show_labels=True, show_conf=True, vid_stride=1, line_width=None, visualize=False, augment=False, agnostic_nms=False, classes=None, retina_masks=False, boxes=True, format=torchscript, keras=False, optimize=False, int8=False, dynamic=False, simplify=False, opset=None, workspace=4, nms=False, lr0=0.01, lrf=0.01, momentum=0.937, weight_decay=0.0005, warmup_epochs=3.0, warmup_momentum=0.8, warmup_bias_lr=0.1, box=7.5, cls=0.5, dfl=1.5, pose=12.0, kobj=1.0, label_smoothing=0.0, nbs=64, hsv_h=0.015, hsv_s=0.7, hsv_v=0.4, degrees=0.0, translate=0.1, scale=0.5, shear=0.0, perspective=0.0, flipud=0.0, fliplr=0.5, mosaic=1.0, mixup=0.0, copy_paste=0.0, cfg=None, v5loader=False, tracker=botsort.yaml, **save_dir=runs/segment/train3**

when i train yolov8 segment , in the parameter list， i find **save_dir**,
But on this page， https://docs.ultralytics.com/modes/train/#arguments, i can't find this arguments,
I want to change the storage path after training to a path defined by myself, such as **/pathto/mytrain100**, how should I modify the code or interface.
Is this path completely unchangeable?

**Comment by glenn-jocher** [comment]
@minwim you can use the `project` and `name` arguments to save results to `project/name` directory.

**Comment by minwim** [comment]
thank you!

**Comment by minwim** [comment]
> @minwim you can use the `project` and `name` arguments to save results to `project/name` directory.

it works,thank you

**Comment by glenn-jocher** [comment]
You're welcome @minwim! I'm glad to hear it worked. If you have any more questions or run into any other issues, please don't hesitate to ask. Our team is here to help.
